eclass/cmake.eclass

```diff
@@ -445,6 +451,19 @@ cmake_src_configure() {
 	# Fix xdg collision with sandbox
 	xdg_environment_reset
 
+	local file ver cmreq_isold
+	if ! [[ ${CMAKE_QA_COMPAT_SKIP} ]]; then
+		while read -d '' -r file ; do
+			ver=$(sed -ne "/cmake_minimum_required/I{s/.*\(\.\.\.*\|\s\)\([0-9.]*\)\([)]\|\s\).*$/\2/p;q}" \
+				"${file}" 2>/dev/null \
+			)
+
+			if [[ -n $ver ]] && ver_test $ver -lt "3.5"; then
```

Comment:
I think Ionen remarked on this before, but let's change it to a bash arithmetic test maybe? That way, we don't get the horrible failure if we extract something bad. Or, alternatively, we can maybe check the result against a regex.

But then again the sed seems kind of robust now so maybe it's fine.